src/components/donation/Steps/Submit/Err.tsx

```diff
@@ -1,14 +1,23 @@
+import { SerializedError } from "@reduxjs/toolkit";
+import { FetchBaseQueryError } from "@reduxjs/toolkit/query";
 import Icon from "components/Icon";
 import { GENERIC_ERROR_MESSAGE } from "constants/common";
+import { logger } from "helpers";
+import { useEffect } from "react";
 
 type Props = {
-  msg?: string;
+  error: FetchBaseQueryError | SerializedError | undefined;
 };
-export default function Err({ msg = GENERIC_ERROR_MESSAGE }: Props) {
+
+export default function Err({ error }: Props) {
+  useEffect(() => {
+    logger.error(error);
+  }, [error]);
+
   return (
     <div className="grid place-items-center content-center gap-6 p-4 @md:p-8">
       <Icon type="ExclamationCircleFill" size={60} className="text-red" />
-      <p className="text-center">{msg}</p>
+      <p className="text-center">{GENERIC_ERROR_MESSAGE}</p>
```

Comment:
All uses of this component display the generic error message, so just hardcode it